ext/web/lib.deno_web.d.ts

```diff
@@ -271,6 +271,7 @@ interface AbortSignal extends EventTarget {
   /** Returns true if this AbortSignal's AbortController has signaled to abort,
    * and false otherwise. */
   readonly aborted: boolean;
+  readonly reason?: any;
```

Comment:
Should we use `unknown`?

Reply:
`WritableStream.abort` signature is `abort(reason?: any): Promise<void>;`, so probably best to align with that.

Reply:
I think that makes sense for inputs, but not sure about outputs. It might be best to make this restrictive then we can loosen it later.

Reply:
Ok, I'll change to `unknown`.